doc/model.rst

```diff
@@ -0,0 +1,683 @@
+
+In the `Instructions` subsection, we provide an example illustrating the deployment of a standard model.
+
+Instructions
+------------
+This example provides a demonstration of how to initialize and launch a standard Model
+within the Experiment workflow.
+
+All workflow entities are initialized through the Experiment API. To create a Model,
+the function ``Experiment.create_model()`` is used. To create Model, we must first
+initialize an instance of the Experiment class. Import the SmartSim Experiment module
+and initialize an Experiment instance. Provide a `name` and `launcher`:
+
+.. code-block:: python
+
+    from smartsim import Experiment
+
+    # Init Experiment and specify to launch locally
+    exp = Experiment(name="getting-started", launcher="local")
+
+Models require run settings objects which are initialized through an Experiment instance.
+Use the experiment instance, `exp`, to access the helper method ``Experiment.create_run_settings()``
+to initialize a ``RunSettings`` object. In this example, we use the Python
+executable to execute the application script, "script.py":
+
+.. code-block:: python
+
+    settings = exp.create_run_settings(exe="python", exe_args="script.py")
+
+We now have an instance of run settings name `settings`. Using the instance, instruct SmartSim
+to encapsulate the simulation as a Model through ``Experiment.create_model()``:
+
+.. code-block:: python
+
+    model = exp.create_model(name="example-model", run_settings=settings)
+
+Remember that all entities are launched, monitored and stopped by the Experiment instance.
+Specify the model instance, `model`, to ``Experiment.start()`` to start the standard model:
+
+.. code-block:: python
+
+    exp.start(model)
+
+When the Experiment Python driver script is executed, two files from the standard model will be created
+in the Experiment working directory:
+
+1. `model.out` : this file will hold outputs produced by the Model workload
+2. `model.err` : will hold any errors that occurred during Model execution
+
+-----------------
+A Colocated Model
+-----------------
+A colocated model is when both the application and the
+database share compute resource. To create a colocated model,
+users first initialize a Model instance with the ``Experiment.create_model()`` function.
+A user must then colocate the model with the Model API function ``Model.colocate_db()``.
+This instructs SmartSim to launch an Orchestrator on the same compute
+nodes as the model when the model instance is deployed via ``Experiment.start()``.
+
+There are **three** different Model API helper functions to colocate a Model:
+
+- ``Model.colocate_db()``: An alias for `Model.colocate_db_tcp()`.
+- ``Model.colocate_db_tcp()``: Colocate an Orchestrator instance with this Model over TCP/IP.
+- ``Model.colocate_db_uds()``: Colocate an Orchestrator instance with this Model over UDS.
+
+Each function will initialize an unsharded database to the Model instance. Only the partnered Model
```

Comment:
The "to the Model" of `Each function will initialize an unsharded database to the Model instance` is confusing. What does "initializing an unsharded database to the Model instance" mean?

Consider "Each function initializes an unsharded database accessible only to the model"

Consider avoiding "partnered" in favor of "colocated" to avoid mixing terminology

Reply:
ahhh I see